backend/src/bins/start_init.rs

```diff
@@ -38,6 +38,12 @@ async fn setup_or_init(cfg_path: Option<PathBuf>) -> Result<(), Error> {
         .arg("/usr/local/bin/aptitude")
         .invoke(crate::ErrorKind::OpenSsh)
         .await?;
+    Command::new("ln")
```

Comment:
Put this in postinst instead. With `/usr/bin` instead of `/usr/local/bin`

Reply:
done

Reply:
done